kolibri/plugins/coach/assets/src/views/reports/LearnerExerciseDetailPage/AttemptSummary.vue

```diff
@@ -79,7 +79,7 @@
     $trs: {
       statusMastered: 'Completed',
       statusInProgress: 'In progress',
-      requirementsMOfN: 'Completion: {m ,number} out of {n, number} correct',
+      requirementsMOfN: 'Completion: {correct, number} correct out of last {total, number}',
```

Comment:
I confess that my suggestion was partly motivated by the desire to get rid of the word "Completion", but you managed to bring it back in. For some reason I'm having trouble finding the satisfying translation in any of the languages I'm generally able to... :confused:

Reply:
Okay, please work with @khangmach and @jtamiace on a better string for 0.12

In the meantime, I'm closing this PR

Reply:
Agreed, @khangmach @jtamiace let's open the list for the next round of string refactoring/audit :slightly_smiling_face: